build.gradle

```diff
@@ -244,7 +250,7 @@ task copytest(type : Copy) {
 
             })
     from tasks.getByPath('compileJava').getClasspath().filter(
-            { f -> f.getName().endsWith(".jar") && !f.getName().equals("rt.jar") && !f.getName().equals("tools.jar")
+            { f -> f.getName().endsWith(".jar") && !f.getName().equals("rt.jar") && !f.getName().equals("tools.jar") && !f.getName().equals("junit-4.10.jar")
```

Comment:
Jason has debugged this some more: the json-simple-1.1.1 dependency downloads junit-4.10.
Then
https://github.com/typetools/checker-framework-inference/blob/master/tests.xml#L23
included all jar files in alphabetical ordering.
Because of some change in annotation-tools, now suddendly junit-4.10.jar is the first jar containing an old class, which caused test failures.
This line filters out this dependency and makes the test pass.